src/models/thelio-mega-r1.0/repairs.md

```diff
@@ -0,0 +1,538 @@
+7. If utilizing a slot for the first time, peel the plastic backing off of the heatsink to expose the double-sided thermal tape for that slot.
+
+![M.2 thermal tape](./img/m2-thermal-tape.jpg)
+
+8. Replace the M.2 heatsinks, GPUs, GPU brace, inner partition, and top case.
+
+## Removing the CPU shroud:
+
+The CPU shroud guides airflow through the CPU coolers. It covers the CPU and RAM slots on the motherboard.
+
+**Tools required:** Cross-head (Phillips) screwdriver (optional)  
+**Time estimate:** 7 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>
+
+### Steps to remove the CPU shroud:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case), [remove the inner partition](#removing-the-inner-partition), and [remove the GPU brace](#replacing-a-gpu).
+2. Unplug the connectors for the two CPU shroud-mounted fans.
+
+![Top exhaust fans](./img/top-fan-connectors.jpg)
+
+3. Unscrew the four back screws holding the CPU shroud in place.
+
+![CPU shroud screws](./img/cpu-shroud-screws.jpg)
+
+5. Pull the CPU shroud away from the machine.
+
+## Replacing the RAM:
+
+Thelio Mega r1.0 supports up to 256GB (8x32GB) of RAM. The RAM sticks are Unregistered/Unbuffered ECC DDR4 DIMMs running at a speed of 3200MHz. If you've purchased new RAM, need to replace your RAM, or are reseating your RAM, follow these steps.
+
+**Tools required:** Cross-head (Phillips) screwdriver (optional)  
+**Time estimate:** 20 minutes  
+**Difficulty:** Medium <span style="color:orange;">●</span>
+
+### Steps to replace the RAM:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case), [remove the inner partition](#removing-the-inner-partition), [remove the GPU brace](#replacing-a-gpu), and [remove the CPU shroud](#removing-the-cpu-shroud).
+2. To remove an existing RAM stick, flip the bottom latch down away from the stick, then pull the stick out of the slot, starting from the bottom edge. (The top of the RAM slot does not move.)
+
+![RAM slots](./img/ram-slots.jpg)
+
+_The CPU cooler has been removed for better visibility in the above photo; removing the CPU cooler is not required to access all eight of the RAM slots._
+
+3. Make sure the tab on the bottom of the slot is open (pulled downwards), then insert the new RAM (or re-seat the existing RAM) into the slot.
+    - The RAM stick will only fit in one direction. On the left side of the CPU, the larger group of pins goes on bottom; on the right side of the CPU, the larger group of pins goes on top.
+    - Use the following guide for placement of the RAM sticks:
+
+![RAM slot order](./img/ram-order.webp)
+
+4. Replace the CPU shroud, GPU brace, inner partition, and top case.
+
+## Removing the top crossbar:
+
+The top crossbar provides a brace for the outer case. It needs to be removed in order to add/remove drives in the top four 2.5" drive slots, and when accessing certain other components (such as the top Thelio-IO board and the CPU cooler-mounted fan.)
```

Comment:
> It needs to be removed in order to add/remove drives in the top four 2.5" drive slots,

Is this still the case? I thought the revised crossbar no longer interfered with the drive screws.